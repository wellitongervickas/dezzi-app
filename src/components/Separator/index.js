import React from 'react';
import SeparatorContainer from 'components/Separator/styles';

const Separator = () => <SeparatorContainer />;

export default Separator;
