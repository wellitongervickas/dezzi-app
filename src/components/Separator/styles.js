import styled from 'styled-components';

import * as sizes from 'assets/stylesheets/js/sizes';

const SeparatorContainer = styled.div`
  height: 2px;
  background-color: #ccc;
  margin: ${sizes.md()} 0;
`;

export default SeparatorContainer;
