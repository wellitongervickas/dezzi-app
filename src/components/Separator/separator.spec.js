import React from 'react'
import { render, cleanup } from '@testing-library/react';
import 'jest-styled-components';

import Separator from '.';

describe('Components Separator', () => {
  afterEach(cleanup);

  it('should have a snapshot with a default Separator component', async () => {
    const { asFragment } = render(<Separator />);
    expect(asFragment(<Separator />)).toMatchSnapshot();
  });
});
