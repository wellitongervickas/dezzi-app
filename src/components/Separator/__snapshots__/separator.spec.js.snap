// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components Separator should have a snapshot with a default Separator component 1`] = `
<DocumentFragment>
  .c0 {
  height: 2px;
  background-color: #ccc;
  margin: 32px 0;
}

<div
    class="c0"
  />
</DocumentFragment>
`;
